notebooks/1.0-vse-eda-rooms.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>If you have some time free it's nice to clean the outlier in this graphic, look this <a href="https://kite.com/python/answers/how-to-remove-outliers-from-a-pandas-dataframe-in-python" target="_blank">link</a> to understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Nice! Probably won't have time to do it today, but i'll definitely analyse it without  the outliers.</p>